nexusutils/dataconverter/readers/em_spctrscpy/utils/em_generic_eln_io.py

```diff
@@ -131,19 +136,19 @@ def parse_user_section(self, template: dict) -> dict:
     def parse_sample_section(self, template: dict) -> dict:
         """Copy data in sample section."""
         # check if required fields exists and are valid
-        print('Parsing sample...')
+        print("Parsing sample...")
         src = "sample"
-        trg = "/ENTRY[entry]/sample/"
+        trg = "/ENTRY[entry" + str(self.entry_id) + "]/sample/"
         assert isinstance(self.yml[src], fd.FlatDict), \
-            'Required section ' + src + ' does not exist!'
+            "Required section " + src + " does not exist!"
         assert isinstance(self.yml[src + ":atom_types"], list), \
             "Facing an ELN schema instance with an incorrect atom_types info!"
         assert len(self.yml[src + ":atom_types"]) >= 1, \
             "Facing an ELN schema instance with an empty atom_types info!"
         for symbol in self.yml[src + ":atom_types"]:
             assert isinstance(symbol, str), \
                 "Facing an atom_types list entry which is not a string!"
-            assert (symbol in chemical_symbols) & (symbol != 'X'), \
+            assert (symbol in chemical_symbols) & (symbol != "X"), \
```

Comment:
Is this a typo or really a bitwise & operator?

Reply:
bitwise & but, yeah maybe and is easier to read, changed it